src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -12,28 +13,50 @@ interface IVedtakProps {
 
 const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ fagsak }) => {
     const [brev, setBrev] = React.useState('Genererer forhåndsvisning...');
+    const [errorMessage, setErrorMessage] = React.useState(undefined);
 
     React.useEffect(() => {
-        hentVedtaksbrev(fagsak.id)
+        hentVedtaksbrev(fagsak)
             .then((response: Ressurs<IVedtaksBrev>) => {
-                setBrev(response.data);
+                if (response.status === RessursStatus.SUKSESS) {
+                    setBrev(response.data);
+                    setErrorMessage(undefined);
+                } else {
+                    setErrorMessage(
+                        response.melding !== undefined
+                            ? response.melding
+                            : 'Kunne ikke generere forhåndsvisning.'
+                    );
+                }
             })
             .catch(error => {
-                setBrev('kunne ikke generere forhåndsvisning. Internal Error: ' + error);
+                const displayed = 'Internal Error: Kunne ikke generere forhåndsvisning.';
+                setBrev(displayed);
+                setErrorMessage(displayed);
             });
     });
 
+    const content =
```

Comment:
Would it be better to move this to the return? I dont see why we need to move this before the return. In any case I would suggest moving it to another component in a different file. 

Also, I think we should use SkjemaFeil or something and not the SkjemaGruppe with a fault. It is syntactically more correct :)

Reply:
What is SkjemaFeil? Cannot find it on design.nav.no, neither can find it in familie-ks-sak-frontend

Reply:
I think AlertStripeFeil will be a nice component here. The one I actually talked about was SkjemaelementFeilmelding.

Reply:
Yes I tried and it worked!